carbon-design-system/carbon-website
# fix(global-search): focus input on render

Addresses an accessibility issue where when the spyglass icon is clicked the text input does not receive focus in the global search component.  ♿

## changes
- adds a GlobalSearchInput component
- replaces `<input>` in GlobalSearch with GlobalSearchInput component

## notes
Had a bit of trouble getting the input to focus. Wound up using a solution suggested by @lovemecomputer that works great making the `.focus()` call asynchronous. If this is problematic I'm happy to keep hacking on it. 👍🏽

## to test
Give the spyglass in the header a click which should open up the text input **that should totally be focused**.
